DDF-1617: renamed anonymous user to 'guest'

ricklarsen opened the pull request:
To test: ensure all i-tests pass, 

* start up, ensure anonymous configuration defaults to "guest"
* sign out of search ui, execute a query, check logs for guest user.

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/codice/ddf/331)
<!-- Reviewable:end -->

ricklarsen commented:
@vinamartin @djblue @clockard @stustison @oscaritoro @gordocanchola

clockard commented:
Should we rename classes like AnonymousAuthenticationToken and AnonymousPrincipal to GuestAuthenticationToken and GuestPrincipal?

clockard commented:
Other than possibly renaming classes LGTM
:+1:

oscaritoro commented:
Reviewed 19 of 19 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

<sup>**[platform/security/idp/security-idp-client/src/test/resources/SAMLResponse-malformedAssertion.xml, line 27 \[r1\]](https://reviewable.io:443/reviews/codice/ddf/331#-K2wkVaQsllQnU3X-ysE)** ([raw file](https://github.com/codice/ddf/blob/c979f727deccad54e573be8d586da458bdac1870/platform/security/idp/security-idp-client/src/test/resources/SAMLResponse-malformedAssertion.xml#L27)):</sup>
Do these need to be "re-signed" after changes?
<!-- Reviewable comment -K2wkVaQsllQnU3X-ysF:-900687033 -->

---

<sup>**[platform/security/idp/security-idp-client/src/test/resources/SAMLResponse-multipleAssertions.xml, line 40 \[r1\]](https://reviewable.io:443/reviews/codice/ddf/331#-K2wkhZpgKiwJ8Jm7VaQ)** ([raw file](https://github.com/codice/ddf/blob/c979f727deccad54e573be8d586da458bdac1870/platform/security/idp/security-idp-client/src/test/resources/SAMLResponse-multipleAssertions.xml#L40)):</sup>
Do these need to be resigned after changing?
<!-- Reviewable comment -K2wkhZpgKiwJ8Jm7VaR:1125508146 -->

---

<sup>**[platform/security/idp/security-idp-client/src/test/resources/SAMLResponse.xml, line 27 \[r1\]](https://reviewable.io:443/reviews/codice/ddf/331#-K2wkoHzF16ZCfbMTC2J)** ([raw file](https://github.com/codice/ddf/blob/c979f727deccad54e573be8d586da458bdac1870/platform/security/idp/security-idp-client/src/test/resources/SAMLResponse.xml#L27)):</sup>
Do these need to be resigned after changing?
<!-- Reviewable comment -K2wkoHzF16ZCfbMTC2K:1125508146 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/331)
<!-- Sent from Reviewable.io -->

djblue commented:
LGTM :+1:

stustison commented on platform/security/sts/security-sts-anonymousclaimshandler/src/main/resources/OSGI-INF/metatype/metatype.xml:
```diff
@@ -17,7 +17,7 @@
 	<OCD description="Anonymous Claims Handler Configuration"
```
Can you also change the description and name to say Guest instead of Anonymous?

ricklarsen replied:
fixed.

ricklarsen replied:
!resolved

oscaritoro commented:
1. Are the role=anonymous instances Ok inside platform/admin/ui/src/test/resources/ getClaimsConfiguration.json and listServices.json?
2. Should the "isAnonymous" in line 79 of ddf/catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/service/UserService.java be changed to "isGuest"?
3. Are the instances of A/anonymous user in the getSubject(String) method of ddf/platform/security/interceptor/security-interceptor-anonymous/src/main/java/org/codice/ddf/security/interceptor/AnonymousInterceptor.java Ok? How about the instance of "anonymous user" in handleMessage(SoapMessage) in the same file?
4. Is the instance of anonymous user in line 222 of ddf/platform/security/security-services-app/src/main/resources/features.xml Ok?
5. Is the instance of anonymous user in line 107 of ddf/platform/security/sts/security-sts-anonymousclaimshandler/src/main/java/org/codice/ddf/security/claims/anonymous/AnonymousClaimsHandler.java Ok?
6. Should the comment in line 95 of ddf/libs/activities/src/test/java/org/codice/ddf/activities/ActivityEventTest.java be updated?
7. Should the comment in line 164 of ddf/libs/notifications/src/main/java/org/codice/ddf/notifications/Notification.java be updated? How about line 36 of ddf/libs/notifications/src/test/java/org/codice/ddf/notifications/NotificationTest.java?
8. Should the comments in lines 117 and 238 of ddf/catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/ActivityController.java be updated?
9. Should the comments in lines 111 and 217 of ddf/catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/NotificationController.java be updated?
10. Should the comment in line 76 of ddf/catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/service/WorkspaceService.java be updated?
11. Should the comment on line 112 of ddf/platform/security/handler/security-handler-anonymous/src/main/java/org/codice/ddf/security/handler/anonymous/AnonymousHandler.java be updated?
12. Should the comments line 22 of ddf/platform/security/handler/security-handler-api/src/main/java/org/codice/ddf/security/handler/api/AnonymousAuthenticationToken.java be updated?
13. Should the instance of anonymous users inside the span tag in ddf/platform/admin/ui/src/main/webapp/templates/installer/anonClaims.handlebars be updated?
14. Should the 2 instances of anonymous user in line 867 of ddf/distribution/docs/src/main/resources/ExtContents.adoc be updated?
15. Should the instances of "anonymous user" and "anonymous access" be updated in ddf/distribution/docs/src/main/resources/ManContents.adoc?
16. Should the instances of "anonymous" in ddf/distribution/docs/src/main/resources/ManContents.adoc be updated?
17. Should "username": "Anonymous", in line 50 of ddf/catalog/ui/search-ui/standard/test.js be updated?

ricklarsen commented:
1. fixed
2. That one should stay "isAnonymous"
3. Those should be ok, I think.
4. I could go either way with that one, so I made it "anonymous/guest"
5. This one's fine
6. I added /guest to that one. 
7. fixed.
8. fixed.
9. fixed.
10. fixed.
11. fixed.
12. I left this one.
13. I _think_ these are ok to stay as "anonymous"
14. I think this one is correct.
15. a few. fixed.
16. some should. fixed.
17. fixed.

oscaritoro commented:
Latest one lgtm :+1:

pklinef commented on distribution/docs/src/main/resources/ManContents.adoc:
```diff
@@ -2709,7 +2709,7 @@ For example, installing the `security-sts-ldaplogin` feature adds an ldap realm.
 
 |anon(anonymous)
-|provides anonymous access
+|provides anonymous/guest access
```
I think we should just say guest in the documentation instead of anonymous/guest.

ricklarsen replied:
!fixed

pklinef commented on platform/security/platform-security-core-api/src/main/java/ddf/security/principal/AnonymousPrincipal.java:
```diff
@@ -18,11 +18,11 @@
- * Principal that designates a {@link ddf.security.Subject} as anonymous
+ * Principal that designates a {@link ddf.security.Subject} as guest
  */
 public class AnonymousPrincipal implements Principal {
```
Is there a reason why we did not rename classes to use guest instead of anonymous?

ricklarsen replied:
I believe there are, but perhaps @stustison can explain them somewhat better than I can.

stustison replied:
No reasons that I can think of. I believe we weren't renaming classes simply because we know that anonymous = guest, but we certainly could rename them ¯\_(ツ)_/¯

ricklarsen replied:
!resolved

pklinef commented on platform/security/handler/security-handler-anonymous/src/main/java/org/codice/ddf/security/handler/anonymous/AnonymousHandler.java:
```diff
@@ -71,7 +71,7 @@ public HandlerResult getNormalizedToken(ServletRequest request, ServletResponse
         HandlerResult result = new HandlerResult();
```
The auth type should be changed from ANON to guest.  Making it clear to admins that this authentication type is for guests was one of the main goals for renaming.

ricklarsen replied:
fixed.

ricklarsen replied:
!resolved

oscaritoro commented:
Reviewed 13 of 13 files at r2, 5 of 5 files at r4.
Review status: all files reviewed at latest revision, 4 unresolved discussions, some commit checks failed.

---

_[distribution/docs/src/main/resources/ManContents.adoc, line 2712 \[r3\]](https://reviewable.io:443/reviews/codice/ddf/331#-K2xeARLziFMeUS3i5rT-r3-2712:-K3LC9nIPoJJs1DEhjGY:-1283327431) ([raw file](https://github.com/codice/ddf/blob/be567645b3e561bee98657508b92a81af6e6b32a/distribution/docs/src/main/resources/ManContents.adoc#L2712)):_
From another comment it looks like we'll want to change not just description but also the type from anon(anonymous) to guest?

---


_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/331)_
<!-- Sent from Reviewable.io -->

stustison commented:
_[distribution/docs/src/main/resources/ManContents.adoc, line 2712 \[r3\]](https://reviewable.io:443/reviews/codice/ddf/331#-K2xeARLziFMeUS3i5rT-r3-2712:-K3LHyZSfg6fDY9GeE04:765801523) ([raw file](https://github.com/codice/ddf/blob/be567645b3e561bee98657508b92a81af6e6b32a/distribution/docs/src/main/resources/ManContents.adoc#L2712)):_
Basically, anything that a user might see should ONLY say "guest". A user or admin should never see the word "anonymous" anywhere.

---


_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/331)_
<!-- Sent from Reviewable.io -->

ricklarsen commented:
Updated all classes and methods, please re-review....

clockard commented on catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/solr/FilteringDynamicSchemaResolver.java:
```diff
@@ -72,8 +72,8 @@ public String getCaseSensitiveField(String mappedPropertyName) {
     @Override
-    public List<String> getAnonymousField(String field) {
-        List<String> fields = super.getAnonymousField(field);
+    public List<String> getGuestField(String field) {
```
I don't think this is related to the anonymous/guest user.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

clockard commented on catalog/ui/search-ui/search-endpoint/src/test/groovy/org/codice/ddf/ui/searchui/query/solr/FilteringDynamicSchemaResolverTest.groovy:
```diff
@@ -129,7 +129,7 @@ class FilteringDynamicSchemaResolverTest extends Specification {
     def "Filter used anonymous fields that are unknown"() {
         when:
-        resolver.getAnonymousField("title")
+        resolver.getGuestField("title")
```
Same comment as above

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java:
```diff
@@ -341,7 +341,7 @@ public boolean isPrivateField(String solrFieldName) {
      * @return a list of possible Solr field names that match the given field. If none are found,
      *         then an empty list is returned
      */
-    public List<String> getAnonymousField(String field) {
```
This should not be changed, this has nothing to do with security.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java:
```diff
@@ -572,7 +572,7 @@ private String getQuery(String attributeName, List<String> ids) throws IngestExc
 
         StringBuilder queryBuilder = new StringBuilder();
 
-        List<String> mappedNames = resolver.getAnonymousField(attributeName);
```
Revert.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java:
```diff
@@ -764,10 +764,10 @@ private String getMappedPropertyName(String propertyName, AttributeFormat format
      */
     @Override
     public SolrQuery propertyIsNull(String propertyName) {
-        List<String> possibleFields = resolver.getAnonymousField(propertyName);
```
Revert all this.

ricklarsen replied:
fixed.

ricklarsen replied:
!resolved

stustison commented on catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-schema-bindings/src/main/resources/xml/XMLSchema.dtd:
```diff
@@ -243,7 +243,7 @@
                 id          ID                     #IMPLIED
                 %sequenceAttrs;>
 
-        <!-- an anonymous grouping in a model, or
```
Revert.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/solr/FilteringDynamicSchemaResolver.java:
```diff
@@ -72,8 +72,8 @@ public String getCaseSensitiveField(String mappedPropertyName) {
 
     @Override
-    public List<String> getAnonymousField(String field) {
-        List<String> fields = super.getAnonymousField(field);
```
Revert all these.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on catalog/ui/search-ui/search-endpoint/src/test/groovy/org/codice/ddf/ui/searchui/query/solr/FilteringDynamicSchemaResolverTest.groovy:
```diff
@@ -119,7 +119,7 @@ class FilteringDynamicSchemaResolverTest extends Specification {
         when:
         resolver.addFieldsFromServer(server)
-        resolver.getAnonymousField("title")
+        resolver.getGuestField("title")
```
Revert.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClient.java:
```diff
@@ -210,7 +210,7 @@ protected String getSortProperty(QueryRequest request, SolrQuery query) {
                         resolver.getField(Metacard.EFFECTIVE, AttributeType.AttributeFormat.DATE,
                                 false), order);
             } else {
-                List<String> resolvedProperties = resolver.getAnonymousField(sortProperty);
```
Revert.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on distribution/docs/src/main/resources/ExtContents.adoc:
```diff
@@ -2682,7 +2684,7 @@ ddf@local>expand Title "VP-Engineering Manager"
 | /--------\              /-------------\        |   |       |           ^    |              |
 | |        |              |             |        |   |       |           |    |              |
-| |  Dumb  |              |  Anonymous  |        |   |  c444 |           |    v              |
+| |  Dumb  |              |  Guest  |        |   |  c444 |           |    v              |
```
Looks like the formatting got messed up here.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on platform/security/idp/security-idp-client/src/test/resources/SAMLResponse-malformedAssertion.xml:
```diff
@@ -24,4 +24,4 @@
     7C3BreFrsDAfBgNVHSMEGDAWgBThVMeX3wrCv6lfeF47CyvkSBe9xjAgBgNVHREEGTAXgRVsb2Nh
```
I'm not sure we can actually change any of these assertions because it is going to invalidate the signatures.

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

stustison commented on catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java:
```diff
@@ -754,7 +754,7 @@ private String getMappedPropertyName(String propertyName, AttributeFormat format
     /**
      * This builds a "is null" query based on the property name provided.
      *
-     * Since no type is provided with this method, we build a OR chained expression with anonymous field names.
```
This file needs reverted.

ricklarsen replied:
!fixed

ricklarsen replied:
fixed.

stustison commented on catalog/ui/search-ui/standard/test.js:
```diff
@@ -47,7 +47,7 @@ app.listen = function() {
                 "user": {
-                    "username": "Anonymous",
+                    "username": "Guest",
                     "isAnonymous": "true"
```
Didn't you change this to isGuest on the Java side?

ricklarsen replied:
fixed.

ricklarsen replied:
!fixed

vinamartin commented:
Anywhere `an anonymous` is used, it should have been changed to `a guest` not `an guest`.

stustison commented on catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/service/UserService.java:
```diff
@@ -76,8 +76,8 @@ public void getUser(final ServerSession remote, Message message) {
                 if (subject instanceof ddf.security.Subject) {
-                    userMap.put("isAnonymous",
-                            String.valueOf(((ddf.security.Subject) subject).isAnonymous()));
+                    userMap.put("isGuest",
```
It looks like you changed the JSON here, but you didn't change any of the corresponding JS that reads this. This is going to break the Search UI.

ricklarsen replied:
!fixed

oscaritoro commented on platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js:
```diff
@@ -311,7 +311,7 @@ define([
         onRender: function () {
-            this.listItems.show(new AnonClaimsMultiValueCollection({
+            this.listItems.show(new GuestClaimsMultiValueCollection({
                 collection: this.collectionArray
```
new word? (guestymous)

ricklarsen replied:
fixed.

oscaritoro commented:
Reviewed 29 of 140 files at r5, 8 of 11 files at r7.
Review status: 36 of 92 files reviewed at latest revision, 12 unresolved discussions.

---

_[platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js, line 359 \[r7\]](https://reviewable.io:443/reviews/codice/ddf/331#-K3Vk9qH8eJtXpl-2ifV:-K3Vk9qH8eJtXpl-2ifW:1019613529) ([raw file](https://github.com/codice/ddf/blob/be47423d7b1d4692c1eef4c1212edf0f9bc74fef/platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js#L359)):_
guestymous?

---


_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/331)_
<!-- Sent from Reviewable.io -->

stustison commented:
_[platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js, line 359 \[r7\]](https://reviewable.io:443/reviews/codice/ddf/331#-K3Vk9qH8eJtXpl-2ifV:-K3VmQC8ZCmKdDyGoFFO:-2110122599) ([raw file](https://github.com/codice/ddf/blob/be47423d7b1d4692c1eef4c1212edf0f9bc74fef/platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js#L359)):_
Word of the day.

---


_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/331)_
<!-- Sent from Reviewable.io -->

oscaritoro commented:
Reviewed 88 of 140 files at r5, 3 of 11 files at r7.
Review status: all files reviewed at latest revision, 13 unresolved discussions.

---

_[platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js, line 315 \[r7\]](https://reviewable.io:443/reviews/codice/ddf/331#-K3QVOy7a0ce9XMCIoSd-r7-315:-K3Vnbh-8Q7yoBVgYHH8:-512060086) ([raw file](https://github.com/codice/ddf/blob/be47423d7b1d4692c1eef4c1212edf0f9bc74fef/platform/admin/ui/src/main/webapp/js/views/installer/GuestClaims.view.js#L315)):_
This was ReviewNinja :(, doesn't appear to know how to map comments to lines nicely...

---


_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/331)_
<!-- Sent from Reviewable.io -->

oscaritoro commented:
Do we need to also worry about the following:
1. distribution/docs/src/main/resources/ManContents.adoc:and Security STS Anonymous Claims Handler (PID: `ddf.security.sts.guestclaims`).
2. 5 instances of anonymous in platform/admin/ui/src/test/js/wd/InstallationTests.js
3. 18 instances of anonymous in platform/admin/ui/src/test/resources/listServices.json
4. 9 instances of anonymous in platform/security/docs/src/main/resources/IntContents.adoc
5. platform/security/filter/security-filter-login/src/test/resources/good_saml.xml:        xsi:type="xs:string">anonymous</saml2:AttributeValue></saml2:Attribute></saml2:AttributeStatement></saml2:Assertion>
6. platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java:            address.addTextNode(org.apache.cxf.ws.addressing.Names.WSA_ANONYMOUS_ADDRESS);

oscaritoro commented:
Also, do we need to worry about these instances of "anon":
1. 6 instances of "anon" in distribution/docs/src/main/resources/ExtContents.adoc:
2. platform/security/platform-security-core-api/src/main/java/ddf/security/SecurityConstants.java:            .asList(".*_WITH_NULL_.*", ".*_DH_anon_.*");

ricklarsen commented:
Do we need to also worry about the following:
1. distribution/docs/src/main/resources/ManContents.adoc:and Security STS Anonymous Claims Handler (PID: ddf.security.sts.guestclaims).

_FIXED_

2. 5 instances of anonymous in platform/admin/ui/src/test/js/wd/InstallationTests.js

_FIXED_

3. 18 instances of anonymous in platform/admin/ui/src/test/resources/listServices.json

_FIXED_

4. 9 instances of anonymous in platform/security/docs/src/main/resources/IntContents.adoc

_FIXED_

5. platform/security/filter/security-filter-login/src/test/resources/good_saml.xml: xsi:type="xs:string">anonymous/saml2:AttributeValue/saml2:Attribute/saml2:AttributeStatement/saml2:Assertion

_Checking on whether this one can be changed without affecting signatures._

6. platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java: address.addTextNode(org.apache.cxf.ws.addressing.Names.WSA_ANONYMOUS_ADDRESS);

_This one is referring to addressing, so it should stay._
xsi:type="xs:string">anonymous/saml2:AttributeValue/saml2:Attribute/saml2:AttributeStatement/saml2:Assertion_Checking on whether this one can be changed without affecting signatures._6. platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java: address.addTextNode(org.apache.cxf.ws.addressing.Names.WSA_ANONYMOUS_ADDRESS);_This one is referring to addressing, so it should stay._

ricklarsen commented:
1. distribution/docs/src/main/resources/ManContents.adoc:and Security STS Anonymous Claims Handler (PID: ddf.security.sts.guestclaims).
*FIXED*
2. 5 instances of anonymous in platform/admin/ui/src/test/js/wd/InstallationTests.js
ui/src/test/resources/listServices.json
*FIXED*
3. 18 instances of anonymous in platform/admin/ui/src/test/resources/listServices.json
*FIXED*
4. 9 instances of anonymous in platform/security/docs/src/main/resources/IntContents.adoc
*FIXED*
5. platform/security/filter/security-filter-login/src/test/resources/good_saml.xml: xsi:type="xs:string">anonymous/saml2:AttributeValue/saml2:Attribute/saml2:AttributeStatement/saml2:Assertion

6. platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java: address.addTextNode(org.apache.cxf.ws.addressing.Names.WSA_ANONYMOUS_ADDRESS);

*This one is referring to the addressing, so it should stay as "ANONYMOUS"*

ricklarsen commented:
1. 6 instances of "anon" in distribution/docs/src/main/resources/ExtContents.adoc:

_FIXED_

2. platform/security/platform-security-core-api/src/main/java/ddf/security/SecurityConstants.java: .asList(".WITH_NULL.", ".DH_anon.");

_FIXED_

oscaritoro commented:
Reviewed 3 of 12 files at r6, 7 of 7 files at r8.
Review status: all files reviewed at latest revision, 12 unresolved discussions.

---



_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/331)_
<!-- Sent from Reviewable.io -->

clockard commented:
Build and ran all tests successfully again. Manually verified that i can run the install and login to the search ui as guest.
